pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -171,9 +179,11 @@ func (s *NodeController) SyncCloud() error {
 	for _, node := range matches.Items {
 		if _, ok := nodeMap[node.Name]; !ok {
 			glog.Infof("Create node in registry: %s", node.Name)
-			_, err = s.kubeClient.Nodes().Create(&node)
+			createdNode, err := s.kubeClient.Nodes().Create(&node)
 			if err != nil {
 				glog.Errorf("Create node error: %s", node.Name)
+			} else {
+				record.Eventf(createdNode, "created", "node %s created", node.Name)
```

Comment:
I recommend using createdNode.Name at the end-- shouldn't make a difference, though.